e2e/pages/events/attachScannedDocsEvent.page.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
   },
 
   enterScannedDocument(scannedDocument, file) {
+    I.waitForElement('.form-group .button');
```

Comment:
can it be :
I.waitForElement('Add new')?

Reply:
Unfortunately the button couldn't be found by text content. Moved waiting to the function and used it here. :+1: